### PR TITLE
working on plans and policies

### PR DESCRIPTION
**Pull request by KurtStam**

### CODE REVIEW

File: manager/ui/apiman-plugin/plugins/apiman/ts/plan-overview.ts [hunk 1]
```diff
@@ -36,6 +43,19 @@ module Apiman {
     });
     }
 
+    $scope.removePolicy = function(policy) {
+        alert("policyId=" + policy.id);
```
Comment (msavy) [inline manager/ui/apiman-plugin/plugins/apiman/ts/plan-overview.ts]: Are these alerts debug stuff you didn't mean to include?

Comment (KurtStam) [inline manager/ui/apiman-plugin/plugins/apiman/ts/plan-overview.ts]: I will remove them in the last pass.

**Comment by EricWittmann** [comment]
Cannot automatically merge.  Could you rebase and update the PR?

**Comment by KurtStam** [comment]
updated

**Pull request merged**